.rspec

```diff
@@ -1 +1,2 @@
 --colour
+--require spec_helper
```

Comment:
You actually may not want this here.  The point of putting this here (as recommended by the skeleton RSpec 3 generates) is to ensure it's always loaded w/o having to repeat `require 'spec_helper'` at the top of every spec file.  However, it doesn't look like you bothered to remove the `require 'spec_helper'` lines from the spec files.  Also, if you have this line in `.rspec`, it's important that `spec_helper` is really light weight since it will _always_ get loaded.  The `spec_helper` on the project is currently loading heavyweight dependencies like rspec-rails, rails, etc -- but presumably, some spec files aren't going to need those things loaded, and having this required is going to slow down running them.  That's the reason for this note at the top of the generated `spec_helper`:

https://github.com/rspec/rspec-core/blob/v3.2.3/lib/rspec/core/project_initializer/spec/spec_helper.rb#L7-L13

rspec-rails 3 follows this principle by keeping `spec_helper` nice and light weight, with minimal config that you'll always want, and also providing a `rails_helper` that loads rspec-rails and rails and can be required by the files that need rails loaded.

In the short term, if you want to add config like this, I'd suggest making a new `spec/minimal_spec_helper.rb` (or `spec/fast_spec_helper.rb` or whatever), moving the light-weight, non-rails config there (such as `mock_framework = :mocha` and `order = 'random'`), and updating this to require that file.  Or maybe punt on this for now until the `spec_helper` code has been split into `spec_helper` and `rails_helper`.